scripts/check-translations.js

```diff
@@ -0,0 +1,257 @@
+import fs from 'fs'
+import path from 'path'
+
+// regex constants
+const re_explicit = /[$.]t\([`'].+[`']\)/g
+const re_non_explicit = /[.$]t\((?!['`]).+\)/g
```

Comment:
Those regex will not work for all use case, translation can only have parameters !
I think it would be best to use i18n tooling for such use case https://kazupon.github.io/vue-i18n/guide/tooling.html#vue-i18n-extract. It cannot not be resolved by regex, but need a proper code parser.